dropshots/src/main/java/com/dropbox/dropshots/Dropshots.kt

```diff
@@ -146,7 +237,7 @@ public class Dropshots internal constructor(
       if (!recordScreenshots) {
         val outputPath = writeDiffImage(filename, filePath, bitmap, reference, null)
         throw AssertionError(
-          "$name: Test image (w=${bitmap.width}, h=${bitmap.height}) differs in size" +
+          "$filename: Test image (w=${bitmap.width}, h=${bitmap.height}) differs in size" +
```

Comment:
I wonder if it might be better to use `{className}.{testName}` or something here. It wasn't using the filename before because the filename could be a hashed string or something (which is the behavior we use when filenames get too long).